src/ui/Stationery.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+    import type { Stationery } from "../lib/parsing/parsing";
+
+    interface Props {
+        stationery: Stationery;
+    }
+
+    let { stationery }: Props = $props();
+
+    let stationery2d = $derived(
+        URL.createObjectURL(
+            new Blob([stationery.background_2d], { type: "image/jpeg" }),
+        ),
```

Comment:
Ditto for createObjectURL, given that they don't seem to get revoked anywhere. Pretty sure this is a memory leak.